### PR TITLE
Fix onesegment routes for case start segment is reverse of finish segment

### PR DESCRIPTION
**Pull request by tatiana-kondakova**

### CODE REVIEW

File: routing/index_graph_starter.cpp [hunk 1]
```diff
@@ -295,7 +295,12 @@ void IndexGraphStarter::AddEnding(FakeEnding const & thisEnding, FakeEnding cons
 
   map<Segment, Junction> otherSegments;
   for (auto const & p : otherEnding.m_projections)
+  {
     otherSegments[p.m_segment] = p.m_junction;
+    // We use |otherEnding| to generate propper fake edges in case both endings have projections
```
Comment (bykoianko) [inline routing/index_graph_starter.cpp]: proper

Comment (tatiana-kondakova) [reply routing/index_graph_starter.cpp]: fixed

File: routing/index_graph_starter.cpp [hunk 2]
```diff
@@ -295,7 +295,12 @@ void IndexGraphStarter::AddEnding(FakeEnding const & thisEnding, FakeEnding cons
 
   map<Segment, Junction> otherSegments;
   for (auto const & p : otherEnding.m_projections)
+  {
     otherSegments[p.m_segment] = p.m_junction;
+    // We use |otherEnding| to generate proper fake edges in case both endings have projections
+    // to the same segment. Direction of p.m_segment does not metter.
```
Comment (mpimenov) [inline routing/index_graph_starter.cpp]: s/metter/matter/

Comment (tatiana-kondakova) [reply routing/index_graph_starter.cpp]: done

**Comment by tatiana-kondakova** [comment]
тест: https://github.com/mapsme/omim/pull/7131

**Pull request merged**